socorro/signature/rules.py

```diff
@@ -577,7 +580,7 @@ def action(self, crash_data, result):
 
         signature, notes, debug_notes = self.c_signature_tool.generate(
             signature_list,
-            crash_data.get("crashing_thread"),
+            crashing_thread,
```

Comment:
Earlier we figured out the crashing thread and it defaults to 0, so we should use that instead of figuring it out again (poorly).